web/wp-content/mu-plugins/alter-wpcfm-config-path.php

```diff
@@ -0,0 +1,32 @@
+<?php
+/**
+ * Plugin Name: Alter-wpcfm-config-path
```

Comment:
I'm actually in favor of removing `wp-cfm` and only keeping the Pantheon specific plugins (PHP Sessions and Advanced Page Cache). Any objection @stevector